src/config.js

```diff
@@ -25,10 +24,10 @@ if (!cfg) {
 }
 cfg.file = configPath;
 
-cfg.privateKey = (secret) => {
-//    console.debug(`Using key '${cfg.wallet.address}' for signing.`);
-    const keyObject = keythereum.importFromFile(cfg.wallet.address, path.resolve(homedir, '.nahmii'));
-    return prefix0x(keythereum.recover(secret, keyObject).toString('hex'));
+cfg.privateKey = async (secret) => {
+    const keystore = fs.readFileSync(path.join(homedir, '.nahmii', 'keystore', cfg.wallet.address));
```

Comment:
I reckon we should support the Ethereum keystore file name convention of prefix `UTC--<created_date_time>--` ahead of the address and without the `0x` prefix on the address. What do you think, @katat ?

Reply:
Good catch 👍 

Now it is ready for another round of review.